platform_versioned_docs/version-24.1/launch/advanced.mdx

```diff
@@ -27,6 +22,58 @@ For example, modify the **manifest** section to give the pipeline a name and des
     }
 ```
 
+#### Nextflow configuration order of priority
+
+When launcing pipelines in Platform, Nextflow configuration values can be supplied from the `nextflow.config` file in the pipeline repository and the **Nextflow config file** field in the pipeline launch form. If different values of the same configuration parameter are defined, Nextflow parameters defined in the launch form **Nextflow config file** field override the same parameters in your `nextflow.config` file. 
```

Comment:
So it looks like by default, the repo values are used, unless
- the user specifies an override globally for the compute env
- the user specifies an override in the launch form, possibly overriding what is set for the compute env

I wonder if a table might help with clarity here? Overrides are always kind of complex to explain. Reminds me of overrides for Opscode Chef automation application.

Reply:
@jason-seqera does the example below illustrate the order clearly? While a table may work better, columns or rows for each scenario could be long and unwieldy.

Reply:
@llewellyn-sl, I had to-read it to understand, it looks like I do (?) how it works. But it might be easier to parse if it's presented in an alternate format. Maybe NF users are familiar with this enough that it is clear as is.